pyvrp/read.py

```diff
@@ -383,6 +383,7 @@ def _vehicle_types(self) -> list[VehicleType]:
                 # The literature specifies depot time windows. We do not have
                 # depot time windows but instead set those on the vehicles.
                 tw_early=time_windows[depot_idx][0],
+                start_late=time_windows[depot_idx][1],
```

Comment:
We don't need to pass this right? The default already does the right thing. So there should be no need to modify this file.